contracts/Whitelist.sol

```diff
@@ -20,13 +37,25 @@ contract Whitelist is Ownable {
         address indexed collateral
     );
 
+    ///@notice emitted when Otoken Factory module whitelist an otoken
```

Comment:
`///@notice emitted when the Otoken Factory module whitelists an otoken`

Reply:
so when are we using /// @notice ?

Reply:
for events and variables!